src/assets/svgs/banner.vue

```diff
@@ -0,0 +1,12 @@
rjQ+/+dbUMaDh2AMAgAgFAEBDy7ZvL9wUwJY1q+KGNatSx4CGcvTIC6kjAJADCgCARrdkSeoEE/bbb7wxWgpXXQAA5JsCAKDBZVdfXbgNAddcsCRuu2Jd6hjQUJ54/Kl66gwApFWsV4QATM7q1akTTNh7b9mSOgI0FBsBAqAAAGgC2SWXZDF7duoYE3LpsvPimtUrUseAhmEjQAAUAABNIuvpKdyq+vds25Q6AjSMoaHDqSMAkJgCAKCZXHxx6gQTsn3t6pjX3Zk6BjQEBQAACgCAJpJdemkW8+aljjFuba2tceNll6aOAQ1haNASAIBmpwAAaDLZli1ZtLWljjFut2xYkzoCNAQTAAAoAACaUHbrrYXZD+CKi5anjgANwSkAACgAAJrVypWpE4zL/O6uWL5wfuoYUHhOAQBAAQDQpLK1a7NYtCh1jHFZtWRh6ghQeIcsAQBoegoAgCaWbdqURXd36hhnNbujI3UEKLxyuZI6AgCJKQAAmly2fXsWOb/B7u4spY4ADaG3d289dQYA0lEAABDZTTfl+mSAoWPDqSNAQxgcGEodAYCEFAAARMTJkwFaW1PHOK391i7DlBgYVAAANDMFAAA/ld12W+4mAY4Mj8TuvsHUMaAhDJgAAGhqCgAAXiK79dYsSvlZc/+1h5+ISrWWOgY0hKFBRwECNDMFAACvkN18cxZz5qSOEZVqLb74/ftTx4CG0W8CAKCpKQAAOK1s69Ysli5NmuHPv/G9eOb5gdSXAhqGTQABmpsCAIBXlW3cmGVveENWqc38CP43H90Vn/32PakvATQUewAANDcFAABntW/FRXFf77Mz9nh/d98j8aG//NuoJigeoJENOgUAoKllqQMAUAw33/S2+vqli+KDd94SKxbNn5bHGHjheHzy/3wn/uGBx6Jer6d+ytBwsiyLh374La//AJpUvs56AiC3Nl23Me766rfin3btjjuuXh/v7rkmLl123pR87cEXjsff3vdIfO7bP4hjI6Opnyo0LMUaQHNTAAAwLls2XxN3ffVbUR6rxpd3/jC+vPOHcd0lF8UbrloX29ddHIvnzJ7Q1zs+Wo77e5+Nv7//0fjOY0856g9myOOP76qvW7fGFABAE/LNH4Bxu/KKm0779mGWZbHuwmVx+fKlsXrp4li9ZFHM6eqMrva2aG9ri5FKJQ4cPhrPH3khnu0/FPf3PhtP7O+zxr8A5s+fF4cPH0kdgyn0p5/6w9i2bbPXgABNyAQAAOO2evVF0du79xUfr9fr8aN9B+JH+w6kjsgU+91//4H40Ac/FibHG4ejAAGal1MAABi3zVuuSR2BGXbllZfHO9/15tQxmEIDg4dSRwAgEQUAAOO2ZfO1qSOQwAc+8Otx4YoLUsdgijgKEKB5KQAAGLftO7Zk7e1WjzWbzs7O+PjHPxyZVeMNYXDABABAs1IAADAhV155eeoIJLBx4wZLARpE/8Bg6ggAJKIAAGBCNm+xDKBZWQrQGGwCCNC8FAAATMiWzTYCbFaWAjSGAQUAQNNSAAAwIes3rMvmz5+XOgaJWApQfMePD6eOAEAiCgAAJuy66zamjkBC73//v4zly5eljgEATJACAIAJ27zFMoBm1t3dFb//Bx+xFKDAHnzgkXrqDADMPAUAABP29rff4davyVkKUGx9/QOpIwCQgAIAgElZuXJF6ggk5lSA4rIRIEBzUgAAMCk33LApdQQS6+zsjI999IOWAhRQf58JAIBmpAAAYFK29lyXOgI5cO2mqywFKKB+EwAATUkBAMCk9PRcn3V1daaOQQ5YClA8/fYAAGhKCgAAJu3aa69KHYEc6OzsjI9//MOWAhTIQL8JAIBmpAAAYNJ6LAPgJKcCFIsJAIDmpAAAYNJ+4d1v9Z4vP2UpQHEcPXosdQQAElAAAHBOVq16TeoI5ISlAACQbwoAAM7Jtm3Xp45AjlgKUBwPPvBIPXUGAGaWAgCAc7K1RwHAS1kKUAx99gEAaDoKAADOyebN12SzZnWnjkGOWApQDP39g6kjADDDFAAAnLPrrtuYOgI5YylA/g0oAACajgIAgHPWYxkAp2EpQL71DwyljgDADFMAAHDOfv4ddxr25hU6OzvjYx/9oKUAOdVvDwCApqMAAGBKrF17SeoI5NC1m66yFCCnBvpNAAA0GwUAAFNix44bUkcgpywFyCcTAADNpy11AGg2P/rRU/XeZ55NHQOm3IEDz0epvTN1DKbYN79xd8yZM+ucv84dP/f6+Oxn/ioirAfIi5EXx+JvvnxXvb29PXUUmtQdd97iGwLMMP/oYIb96Z/8Rf1Lf/2V1DEAAJL6/F/+UVy0crn7EZhBJgBghq1dd0m84fYdqWPAtBgZGYlqtZY6BlOou7t7yjbxq1ZrMTIykvopcYqurq5oaXH/RRpu/gEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABrT/wdzwCi7ZAWuMwAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0xMi0wNVQyMTowOTo1MiswMDowMG+syzwAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMTItMDVUMjE6MDk6NTIrMDA6MDAe8XOAAAAAAElFTkSuQmCC"/>
```

Comment:
what kind of svg is this?

Reply:
![image](https://user-images.githubusercontent.com/43110334/147085605-78690a63-3e0f-4c23-8f0a-71303469e07e.png)